src/test/groovy/edu/ie3/simona/model/grid/RefSystemTest.groovy

```diff
@@ -47,28 +47,28 @@ class RefSystemTest extends Specification {
     RefSystem from = RefSystem.apply(Quantities.getQuantity(60d, MEGAVOLTAMPERE), Quantities.getQuantity(110d, KILOVOLT))
     RefSystem to = RefSystem.apply(Quantities.getQuantity(40d, MEGAVOLTAMPERE), Quantities.getQuantity(110d, KILOVOLT))
     Quantity<Dimensionless> impedance = Quantities.getQuantity(0.1d, PU)
-    Quantity<Dimensionless> expected = Quantities.getQuantity(0.15d, PU)
+    Quantity<Dimensionless> expected = Quantities.getQuantity(0.0666d, PU)
 
     when:
     Quantity<Dimensionless> actual = RefSystem.transferImpedance(impedance, from, to)
 
     then:
     actual.getUnit() == impedance.getUnit()
-    testTolerance < Math.abs(actual.getValue().doubleValue() - expected.getValue().doubleValue())
+    Math.abs(actual.getValue().doubleValue() - expected.getValue().doubleValue()) < testTolerance
   }
 
   def "An dimensionless admittance is transferred correctly between to reference system"() {
```

Comment:
"between to" looks like a slip-up... Could be "between reference systems". Would you correct this, also in the test title above?